.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,73 @@
+name: Bump versions for extension only
```

Comment:
What is the file name of this one about? `1_2_b_...`?

Reply:
This file covers both 1, 2 steps and it is a B variant i.e. another variant of the same, specifically for extension only

Reply:
Ok.